test/plugins/common.ts

```diff
@@ -196,6 +196,15 @@ function hasContext() {
   return !!cls.getRootContext();
 }
 
+function notNull<T>(arg: T | null) : T {
+  assert.notStrictEqual(arg, null);
```

Comment:
This seems redundant with the check below.

Reply:
Yep, the below error will never be thrown. It's only there for type safety; I'll add a comment to describe this.

FYI, I think we should prefer an error thrown by the `assert` module, which is why I didn't bother elaborating on the error object thrown below.